create_patched_sdk.py

```diff
@@ -105,358 +69,457 @@ def removeDirectory(dir):
     return True
 
 def main(argv):
-    archs = ARCHS
-    dirs = DIRS
-    targets = TARGETS
-    version = VERSION
+    home_path_env = os.environ.get("HOME")
+    if home_path_env is None:
+        print("The environment-variable $HOME wasn't found")
+        print("Run 'export HOME=<your home-folder path here>' before running this script")
 
-    theos_path = os.environ.get("THEOS")
-    if theos_path is None:
-        print("Environment Variable THEOS does not exist.")
-        print("Please export the environment variable and re-rerun this program")
+        exit(1)
+
+    home_path = Path(home_path_env)
+    if not home_path.exists() or not home_path.is_dir():
+        print("No directory exists at path of environment-variable $HOME")
+        print(f"The path of environment-variable $HOME is {home_path}")
 
         exit(1)
 
-    output_path = os.path.join(theos_path, "sdks")
+    theos_path_env = os.environ.get("THEOS")
+    if theos_path_env is None:
+        print("The path of environment-variable $THEOS doesn't exist.")
+        print("Please run 'export THEOS=<theos path here>' before running this program")
 
-    tbd_path = TBD_PATH
-    xcode_path = XCODE_PATH
+        exit(1)
+
+    theos_path = Path(theos_path_env)
+    if not theos_path.exists() or not theos_path.is_dir():
+        print("No directory exists at path of environment-variable $THEOS")
+        exit(1)
+
+    output_path = Path(theos_path, "sdks")
```

Comment:
`Path` concatenation is usually done with `/`:

`output_path = theos_path / "sdks"`